javascript-practice/src/pages/dashboard.html

```diff
@@ -1,11 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Dashboard</title>
-</head>
-<body>
-  
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <title>Dashboard</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="header-dashboard flex">
+        <div class="header-dashboard-name-box flex-column">
+          <h2 class="header-dashboard-name">Hello, Dat</h2>
```

Comment:
For fake data, make it common. Update avatar pls

Reply:
I'll update it in the next task

Reply:
Use common data pls. Like Hello User